modules/gsp-cluster/service-operator.tf

```diff
@@ -57,6 +57,16 @@ data "aws_iam_policy_document" "service-operator" {
     ]
   }
 
+  statement {
+    actions = [
+      "s3:*",
+    ]
+
+    resources = [
+      "arn:aws:s3:::${var.cluster_name}-*",
```

Comment:
This will presumably allow it to manage the buckets themselves, but not the contents. That's probably what we want...but deleting non-empty s3 buckets is troublesome (if memory serves). Just something to think about

Reply:
does `/` stop the effect of the `*`? ... I thought this would allow ALL things prefixed with cluster-name (although now I think about it, I'm not sure if this is valid)... I'll test in sandbox, one sec

Reply:
You can ignore my comment...
https://docs.aws.amazon.com/AmazonS3/latest/dev/s3-arn-format.html

Reply:
well it didn't work for other reasons anyway :disappointed:  ... changed it to full permisison as per the rds/sqs ones or cloudformaiton will bork with errors